libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -140,21 +146,30 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
    * @deprecated
    */
   @Input()
-  public set tabindex(value: number) {
+  public set tabindex(value: number | undefined) {
     console.warn(
       'The sky-radio `tabindex` property is deprecated. Please use the `tabindex` property on the sky-radio-group component.'
     );
-    this._tabindex = value;
-    this.changeDetector.markForCheck();
+    if (value) {
+      this.#_tabindex = value;
+    } else {
+      this.#_tabindex = 0;
+    }
+    this.#changeDetector.markForCheck();
```

Comment:
Why is this needed (since this is an input)?

Reply:
Because the radio group component calls the input in typescript and, as we discovered in flex the other day, that type of assignment on an input does not trigger change detection. It is only when inputs are updated through template bindings that trigger change detection.

Reply:
Ah, right.